documentation.html

```diff
@@ -0,0 +1,169 @@
+    attempts to
+    cover the basics of how Nu works internally, to get a solid understanding. You will learn how data is treated, what
+    kind
+    of data types Nu supports and how you can write plugins for it.
+</p>
+<h2>Cookbook</h2>
+<p>
+    <a href="https://www.nushell.sh/cookbook/">Nu Cookbook</a> is a collection of examples to help you get the most out
+    of using
+    Nushell. It offers multiple ways of expressing the same pipelines so you can become familiar with all the commands.
+</p>
+<h1>Quick command references</h1>
+
+<style>
+    .flex-grid {
+        display: flex;
+        flex-wrap: wrap;
+        align-items: center;
+    }
+
+    .item {
+        border: 2px solid black;
+        border-radius: 25px;
+        margin: 5px;
+        display: flex;
+        justify-content: center;
+        min-width: 200px;
+        flex-shrink: 0;
+    }
+
+    .stack {
+        padding: 10px;
+        display: flex;
+        flex-direction: column;
+    }
+
+    .row {
+        display: flex;
+        flex-direction: row;
+    }
+
+    .row__item {
+        margin-left: 10px;
+    }
+
+    .invisible {
+        visibility: hidden;
+    }
+
+    .hidden {
+        display: none;
+    }
+
+    #sidebar {
+        min-width: 200px;
+        max-height: 600px;
+        overflow-y: scroll;
+    }
+
+    .frame pre {
+        white-space: pre-wrap;
+    }
+
+    .frame {
+        width: 80%;
+    }
+
+    .button {
+        background: none;
+        color: inherit;
+        border: 1px black solid;
+        border-radius: 10px;
+        padding: 2px;
+        margin: 2px;
+        font: inherit;
+        cursor: pointer;
+        outline: inherit;
+    }
+
+    .grow {
+        flex-grow: 1;
+    }
+</style>
+<div class="container">
+    <div class="row">
+        <div id="sidebar" class="stack">
+            {% for command in site.data.commands %}
+            <div class="row">
+                <button class="row__item button grow"
+                    onclick="toggleFrame('{{ command.name | replace: ' ', '_' }}-usage')">{{ command.name }}</button>
+                {% if command.subcommands %}
+                <button class="row__item button" onclick="toggleVisible('{{ command.name }}-subs')"><i
+                        class="fa fa-plus"></i></button>
+                {% else %}
+                <button class="row__item button invisible"><i class="fa fa-plus"></i></button>
+                {% endif %}
+            </div>
+            <div id="{{ command.name }}-subs" class="hidden">
+                <ul>
+                    {% for sub_command in command.subcommands %}
+                    <li class="row">
+                        <button class="row__item button grow"
+                            onclick="toggleFrame('{{ sub_command.name | replace: ' ', '_' }}-usage')">{{ sub_command.name }}</button>
+                    </li>
+                    {% endfor %}
```

Comment:
I think subcommands should just be shown on the same level as the other commands, without having to click an "expand" button to see them. I'm thinking the fewer clicks one has to make, the better. It also will be easier to work with if subcommands ever have subcommands, which have subcommands... 😅 

What do you think?

Reply:
Yeah, that makes sense. There's just a lot of sub-commands.
But I'm good with either and can change it

Reply:
> There's just a lot of sub-commands.

Yeah let's keep it simple for now and not hide them by default, and return to this decision if things get out of hand.